frontend/app/src/pages/main/v1/workflows/$workflow/components/trigger-workflow-form.tsx

```diff
@@ -63,9 +63,21 @@ export function TriggerWorkflowForm({
   const [cronExpression, setCronExpression] = useState<string>('* * * * *');
   const [cronName, setCronName] = useState<string>('');
 
-  const [selectedWorkflowId, setSelectedWorkflowId] = useState<
-    string | undefined
-  >(defaultWorkflow?.metadata.id);
+  const [selectedWorkflowId, setSelectedWorkflowId] = useState(
+    defaultWorkflow?.metadata.id,
+  );
+
+  const handleClose = useCallback(() => {
+    onClose();
+    setInput('{}');
+    setAddlMeta('{}');
+    setErrors([]);
+    setSelectedWorkflowId(defaultWorkflow?.metadata.id);
+    setTimingOption(defaultTimingOption);
+    setScheduleTime(new Date());
+    setCronExpression('* * * * *');
+    setCronName('');
+  }, [onClose, defaultWorkflow, defaultTimingOption]);
```

Comment:
This is a hack - we should really use a form here but since we're ripping this out soon anyways I think it's fine as-is